lib/index.js

```diff
@@ -206,6 +228,17 @@ export class Baker extends EventEmitter {
 
     // if an object of custom nunjucks filters was provided, add them now
     if (nunjucksFilters) {
+      if (crosswalkPath && nunjucksFilters.prepareCrosswalk) {
```

Comment:
Maybe we should remove crosswalkPath entirely, since it will be unusable for web component embeds. It was added in this PR so there should be no regression

Reply:
as long as we're ok with `window.CROSSWALK` paths to be relative instead of absolute